### PR TITLE
#56 Make master has all the dev feature

### LINKED ISSUE #56: Make master-dev branch has all the dev and master features
Tag existing master branch and merge dev to master

### PR DESCRIPTION
**Pull request by man007yadav**
Fixes #56 Changes -
1. Tagged the existing master branch
2. Added the dev branch features to the master branch

### CODE REVIEW

File: src/elasticsearch/elasticsearchutils.py [hunk 1]
```diff
@@ -31,7 +32,10 @@
   }
  },
  "script" : {
-  "inline" : "ctx._source.status='<STATUS>' ; ctx._source.modified_at='<MODIFIED_AT>'"
+  "params" : {
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py:36]: Can you please remove all formate change or unnecessary change.  Did you add test case ?

File: src/elasticsearch/elasticsearchutils.py [hunk 2]
```diff
@@ -43,22 +47,28 @@
...
+    "status": '<STATUS>',
+    "modified": '<MODIFIED_AT>',
+    "info": '<INFO>',
+    "reason": '<REASON>'
+  }
  }
 }
- 
+
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py]: why this line here?

File: src/elasticsearch/elasticsearchutils.py [hunk 3]
```diff
@@ -43,22 +47,28 @@
...
 payload_search_id = {
     "query": {
-             "match" : { 
+             "match" : {
                 "id" : "<ID>"
             }
     }
-} 
+}
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py]: ?

File: src/elasticsearch/elasticsearchutils.py [hunk 4]
```diff
@@ -77,15 +87,15 @@
     "bool": {
       "must": [
         {
-          "match":{  
+          "match":{
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py]: ?

File: src/elasticsearch/elasticsearchutils.py [hunk 5]
```diff
@@ -77,15 +87,15 @@
...
              "status" : "<STATUS>"
           }
         }
       ],
       "filter": [
         {
-          "range":{  
-               "modified_at":{  
+          "range":{
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py]: ?

File: src/elasticsearch/elasticsearchutils.py [hunk 6]
```diff
@@ -131,14 +141,49 @@
             }
     ]
 }
- 
- 
+
```
Comment (pzou1974) [inline src/elasticsearch/elasticsearchutils.py]: ?

File: src/elasticsearch/elasticsearchutils.py (hunk 1 above)
Comment (man007yadav) [reply src/elasticsearch/elasticsearchutils.py:36]: I havent added any new test case. These are refactoring code changes from dev branch.

File: src/elasticsearch/elasticsearchutils.py (hunk 2 above)
Comment (man007yadav) [reply src/elasticsearch/elasticsearchutils.py]: I think this is a good change to remove unnecessary whitespace from the current code.

File: src/elasticsearch/elasticsearchutils.py (hunk 3 above)
Comment (man007yadav) [reply src/elasticsearch/elasticsearchutils.py]: same as above.

File: src/elasticsearch/elasticsearchutils.py (hunk 4 above)
Comment (man007yadav) [reply src/elasticsearch/elasticsearchutils.py]: same as above.

**Review by pzou1974 — COMMENTED** [review]
I still find a lot of space and line feed.

**Comment by pzou1974** [comment]
This is also wrong.   what I want your help is purely from dev to main and make small change for conflict.
